environment-dev.yml

```diff
@@ -6,48 +6,45 @@ dependencies:
 
   # required dependencies
   - pandas>=2,<2.1
-  - numpy>=1.18.5
-  - fsspec
-  - packaging
-  - psutil
+  - numpy>=1.20.3
+  - fsspec>=2021.07.0
+  - packaging>=21.0
+  - psutil>=5.8.0
 
   # optional dependencies
   # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
   - ray-default>=1.13.0,!=2.5.0
   # https://github.com/modin-project/modin/issues/6336
   - pydantic<2
-  - pyarrow
+  - pyarrow>=7.0.0
   # workaround for https://github.com/conda/conda/issues/11744
   - grpcio!=1.45.*
   - grpcio!=1.46.*
   - dask>=2.22.0
   - distributed>=2.22.0
   # TODO: uncomment after Modin switch to python>=3.9
   # - xarray
-  - Jinja2
-  - scipy
+  - Jinja2>=3.0.0
+  - scipy>=1.7.1
   - s3fs>=2021.8
-  - feather-format
```

Comment:
This is a outdated package, the required functionality is now in `pyarrow`. So I removed it.